packages/demo-gatsby/src/components/bio.js

```diff
@@ -78,11 +78,13 @@ const Bio = () => {
     }
   `)
 
-  const [author] = useLocalJsonForm(data.dataJson, {
+  const [author, authorForm] = useJsonForm(data.dataJson, {
     label: "Author",
     fields,
   })
 
+  usePlugin(authorForm)
```

Comment:
This doesn't seem to be imported from anywhere? Should this be `useFormScreenPlugin`?

Reply:
You're right on both counts! Thanks 😄

Reply:
Thanks again, caught a couple other issues in the demos.

@all-contributors please add @BPScott for review